pipeline/inspection.py

```diff
@@ -109,5 +116,6 @@ def replace_none_with_empty_list(cls, values):
                 values[field] = []
```

Comment:
The errors you are experiencing here might be due to #44. Possible fix in PR #46.